src/core/tests/randomizer.test.js

```diff
@@ -3,15 +3,15 @@ import { buildRandomizer } from "../randomizer";
 
 it("should always return unique message", () => {
   const messages = ["1", "2", "3", "4", "5"];
-  const randomizer = buildRandomizer(messages);
+  const randomizer = buildRandomizer(messages,messages);
```

Comment:
The second argument for `buildRandomizer()` is emoji list. Could you update that? And please, don't forget to add a space here. I know we don't know formatter like Prettier at the moment, sorry.

Something like this:

```js
...
const emojis = [...];
const randomizer = buildRandomizer(messages, emojis);
...
```